.github/workflows/conductor.yml

```
@@ -0,0 +1,217 @@
+      - uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7.0.1
+        with:
+          persist-credentials: false
+      - name: 评论失败原因
+        env:
+          CB_APP_ID: ${{ secrets.CB_APP_ID }}
+          AGENT_APP_SECRET: ${{ secrets.AGENT_APP_SECRET }}
+          ISSUE: ${{ needs.route.outputs.issue }}
+          ROUTE_RC: ${{ needs.route.result }}
+          SPEC_RC: ${{ needs.spec.result }}
+          RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:1:1: TOKEN appears unused. Verify use (or export if used externally)

(shellcheck)